scripts/docs/build-docs.js

```diff
@@ -53,11 +53,11 @@ function runUmberto( options ) {
 		skipLiveSnippets: options.skipLiveSnippets,
 		skipValidation: options.skipValidation,
 		snippetOptions: {
-			production: options.production
+			production: options.production,
+			whitelistedSnippets: whitelistedSnippets ? whitelistedSnippets.replace( '--whitelisted-snippet=', '' ).split( ',' ) : []
```

Comment:
Maybe '--whitelisted-snippet=' should be now rename to '--whitelisted-snippets=' as this option on umberto side now supports array or globs.

Reply:
Can it be simply `--snippets`? We have `--skip-snippets` already. Typing `--whitelisted-snippets` is time-consuming.

Reply:
`--snippets` sounds good.

Reply:
Renamed to `--snippets` for CLI.